File: src/main/java/de/fabilucius/advancedperks/updatechecker/data/UpdateData.java
package de.fabilucius.advancedperks.updatechecker.data;

import java.util.Date;
import java.util.UUID;

public class UpdateData {

    private int downloads;
    private String name;
    private Rating rating;
    private Date releaseDate;
    private int resource;
    private UUID uuid;
    private int id;

    public String getName() {
        return name;
    }

    public Date getReleaseDate() {
        return releaseDate;
    }
}

File: src/main/kotlin/de/fabilucius/advancedperks/updatechecker/data/UpdateData.kt
package de.fabilucius.advancedperks.updatechecker.data

import java.util.Date
import java.util.UUID

data class UpdateData(
    val downloads: Int,
    val name: String,
    val rating: Rating,
    val releaseDate: Date,
    val resource: Int,
    val uuid: UUID,
    val id: Int,
)
